packages/api-reference/src/components/Content/Schema/SchemaDiscriminator.vue

```diff
@@ -90,26 +98,115 @@ const getModelNameFromSchema = (schema: any): string | null => {
   return null
 }
 
+// Get schema with discriminator
+const getSchemaWithDiscriminator = (schemas: any[]) => {
+  return schemas.find((schema: any) => schema.oneOf || schema.anyOf)
+}
+
+const schemaDiscriminators = computed(() => {
+  const schemaDiscriminator = getSchemaWithDiscriminator(value[discriminator])
+
+  if (!schemaDiscriminator) {
+    return value[discriminator]
+  }
+
+  // Get schema with nested discriminators
+  const schemaNestedDiscriminators =
+    schemaDiscriminator.oneOf || schemaDiscriminator.anyOf
+
+  return schemaNestedDiscriminators.map((schema: any) => {
+    if (schema.allOf) {
+      const titledSchema = schema.allOf.find((s: any) => s.title)
+      const referencedSchema = schema.allOf.find((s: any) => !s.title)
+
+      if (titledSchema && referencedSchema) {
+        return {
+          ...titledSchema,
+          properties: {
+            ...titledSchema.properties,
+            ...referencedSchema.properties,
+          },
+          required: [
+            ...(titledSchema.required || []),
+            ...(referencedSchema.required || []),
+          ],
+          oneOf: referencedSchema.oneOf,
+          anyOf: referencedSchema.anyOf,
+        }
+      }
+      return titledSchema || schema
+    }
+    return schema
+  })
+})
+
 // Humanizes discriminator type name e.g. oneOf -> One of
 const humanizeType = (type: string) => {
+  // To do: add a map for the discriminator type names
+  if (type === 'allOf') {
+    const schemaWithDiscriminator = value?.[type]?.find(
+      (schema: any) => schema?.oneOf || schema?.anyOf,
+    )
+    if (schemaWithDiscriminator?.oneOf) {
+      return 'One of'
+    }
+    if (schemaWithDiscriminator?.anyOf) {
+      return 'Any of'
+    }
+  }
+
   return type
     .replace(/([A-Z])/g, ' $1')
     .replace(/^./, (str) => str.toUpperCase())
     .toLowerCase()
     .replace(/^(\w)/, (c) => c.toUpperCase())
 }
+
+// Get current schema
+const discriminatorSchema = computed(
+  () => schemaDiscriminators.value[selectedIndex.value],
+)
+
+// Return current discriminator type
+const discriminatorType = computed(() => {
+  return discriminatorSchema.value?.oneOf ? 'oneOf' : 'anyOf'
+})
+
+// Return current schema's discriminator value
```

Comment:
```suggestion
/* Return current schema's discriminator value */
```

If you use `/* */` it’s shown as a tooltip when hovering the const in other places.

Reply:
I think you need to use `/** */` no?